app/presenters/specialist_document_presenter.rb

```diff
@@ -89,6 +89,10 @@ def show_metadata_block?
     content_item.dig("links", "finder", 0, "details", "show_metadata_block")
   end
 
+  def hide_contents_list?
+    content_item.dig("links", "finder", 0, "details", "hide_contents_list")
```

Comment:
Question: why was this defined as "hide_contents_list" and the metadata one defined as "show_metadata_block"? Why not "show_contents_list"?

Reply:
Regardless I'd change this method to be `show_contents_list?` and return `true` if `content_item.dig("links", "finder", 0, "details", "hide_contents_list")` is `nil` or `false`.